packages/plugin-core/src/commands/Doctor.ts

```diff
@@ -334,10 +414,11 @@ export class DoctorCommand extends BasicCommand<CommandOpts, CommandOutput> {
     return engine;
   }
 
-  addAnalyticsPayload(opts: CommandOpts) {
+  addAnalyticsPayload(opts: CommandOpts, out: CommandOutput) {
     return {
       action: opts.action,
       scope: opts.scope,
+      ...out.extra,
```

Comment:
we should pick what properties we are adding but not extract everything from an `any` parameter